README.md

```diff
@@ -43,17 +43,16 @@ are supported:
 
 ```javascript
 WebVRConfig = {
-
   // Forces availability of VR mode, even for non-mobile devices.
   FORCE_ENABLE_VR: true, // Default: false.
 
   // Complementary filter coefficient. 0 for accelerometer, 1 for gyro.
   K_FILTER: 0.98, // Default: 0.98.
 
-  // How far into the future to predict during fast motion.
-  PREDICTION_TIME_S: 0.050, // Default: 0.050s.
+  // How far into the future to predict during fast motion (in seconds).
+  PREDICTION_TIME_S: 0.040, // Default: 0.040.
```

Comment:
I noticed that the default here was wrong - it's actually [`0.040`](https://github.com/borismus/webvr-polyfill/pull/53/files#diff-c9f2bc69ace9478cfd1156aabd2a8e31L4814)